source/kra/parser.d

```diff
@@ -72,6 +72,8 @@ KRA parseKRAFile(ZipArchive file)
 
 	importAttributes(kra, image.children[0]);
 
+	pointCloneLayers();
```

Comment:
@LunaTheFoxgirl is this the correct placement for finalization?

For reference, can you point me to the part of coding that uses 2-step parsing?

Reply:
Looks like it to me; though I'd recommend calling it `finalizeLayers` so that any other layers that may need a finalization step can be rolled into it as well.